### PR TITLE
Rewrite Dokka-generated markdown to include titles.

### PR DESCRIPTION
**Pull request by swankjesse**
I tried rendering in jekyll mode but that doesn't look as nice with
Mkdocs.

### CODE REVIEW

**Comment by swankjesse** [comment]
Screen Shot 2019-09-28 at 11 51 47 AM

File: deploy_website.sh [hunk 1]
```diff
@@ -30,6 +30,26 @@ cd $DIR
   :okhttp-urlconnection:dokka \
   :okhttp:dokka
 
+# Dokka filenames like `-http-url/index.md` don't work well with MkDocs <title> tags.
+# Assign metadata to the file's first Markdown heading.
+# https://www.mkdocs.org/user-guide/writing-your-docs/#meta-data
+title_markdown_file() {
+  TITLE_PATTERN="s/^[#][#]* *\(.*\)/title: \1 - OkHttp/g"
```
Comment (yschimke) [inline deploy_website.sh]:
```suggestion
  TITLE_PATTERN="s/^[#]+ *\(.*\)/title: \1 - OkHttp/g"
```

Why g? all instances rather than first?

Comment (swankjesse) [reply deploy_website.sh]: Just habit.

Comment (swankjesse) [reply deploy_website.sh]: Oh and I got `+` to work but I needed the `-E` flag because sed is from the 1970s

**Comment by swankjesse** [comment]
Worth it!
Screen Shot 2019-10-06 at 10 47 21 AM

**Pull request merged**